code/game/machinery/lightswitch.dm

```diff
@@ -29,10 +30,22 @@
 	updateicon()
 
 /obj/machinery/light_switch/proc/updateicon()
-	if ((stat & NOPOWER) || buildstage != 2)
+	if(!overlay)
+		overlay = image(icon, "light1-overlay")
+		overlay.plane = LIGHTING_PLANE
+		overlay.layer = ABOVE_LIGHTING_LAYER
```

Comment:
![image](https://user-images.githubusercontent.com/6307265/44965435-e41c7d80-af35-11e8-9531-a306fb823676.png)

Reply:
It's identical to how borg eyes work
![2018-09-03_00-00-03](https://user-images.githubusercontent.com/40967382/44965504-5d06df80-af0c-11e8-8783-35b5a10c8a1a.gif)
You can only see overlays if you can see the base object